lib/sqlalchemy/dialects/mysql/pyodbc.py

```diff
@@ -79,5 +79,18 @@ def _extract_error_code(self, exception):
         else:
             return None
 
+    def on_connect(self):
+        super_ = super(MySQLDialect_pyodbc, self).on_connect()
+
+        def on_connect(conn):
+            if super_ is not None:
+                super_(conn)
+
+            conn.setdecoding(1, encoding='utf-8')  # pyodbc.SQL_CHAR
```

Comment:
can we add a comment linking to https://github.com/mkleehammer/pyodbc/wiki/Unicode#mysql-and-postgresql and also does this work for Py 2.7 ?   it looks like there are different steps

Reply:
Yes, the `setencoding` and `setdecoding` calls are a bit different for Python_2. I could add an `if sys.version_info.major > 2:` block for the current directives and an `else:` with the Python_2 equivalents. Would that be satisfactory?

Reply:
@zzzeek - Done.